### PR TITLE
[CWS] Export cgroup.version

### PR DESCRIPTION
**Pull request by lebauce**
### What does this PR do?

Export the cgroup API version.

### Motivation

This allows narrowing detections to some specific cgroup version.

### CODE REVIEW

File: pkg/security/probe/probe_ebpf.go [hunk 1]
```diff
@@ -698,6 +698,9 @@ func (p *EBPFProbe) setProcessContext(eventType model.EventType, event *model.Ev
 		panic("should always return a process context")
 	}
 
+	// fmt.Printf("Replacing cgroup context %+v with %+v, isResolved: %+v (open: %+v)\n", event.CGroupContext, event.ProcessCacheEntry.CGroup, isResolved, event.Open)
+	// debug.PrintStack()
+
```
Comment (YoannGh) [inline pkg/security/probe/probe_ebpf.go:703]: to remove, otherwise LGTM

**Review by YoannGh — APPROVED** [review]

File: pkg/security/probe/field_handlers_ebpfless.go [hunk 2]
```diff
@@ -160,6 +160,11 @@ func (fh *EBPFLessFieldHandlers) ResolveCGroupID(_ *model.Event, _ *model.CGroup
 	return ""
 }
 
+// ResolveCGroupVersion resolves the version of the cgroup API
+func (fh *EBPFLessFieldHandlers) ResolveCGroupVersion(ev *model.Event, e *model.CGroupContext) int {
```
Comment (YoannGh) [inline pkg/security/probe/field_handlers_ebpfless.go:164]: This function has linter errors about unused arguments

File: pkg/security/probe/probe_ebpf.go (hunk 1 above)
Comment (lebauce) [reply pkg/security/probe/probe_ebpf.go:703]: Ooops. Thanks

**Review by spikat — APPROVED** [review]

**Review by hestonhoffman — APPROVED** [review]

**Review by paulcacheux — APPROVED** [review]

**Pull request merged**